core/offchain/src/api/http.rs

```diff
@@ -983,7 +983,7 @@ mod tests {
 		let (mut api, addr) = build_api_server!();
 
 		for _ in 0..50 {
-			let id = api.request_start("GET", &format!("http://{}", addr)).unwrap();
+			let id = api.request_start("POST", &format!("http://{}", addr)).unwrap();
```

Comment:
Why do we need this?

Reply:
Sending bodies on GET requests requires manually specifying a `Transfer-Encoding` or `Content-Length` header.  These tests do neither.